static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -431,10 +431,15 @@ class IssueRuleEditor extends AsyncView<Props, State> {
   // As more incompatible combinations are added, we will need a more generic way to check for incompatibility.
   checkIncompatibleRule = debounce(() => {
     if (this.props.organization.features.includes('issue-alert-incompatible-rules')) {
-      const incompatibleRule = findIncompatibleRules(this.state.rule);
+      const {conditionIndices, filterIndices} = findIncompatibleRules(this.state.rule);
+      if (conditionIndices !== null || filterIndices !== null) {
+        trackAdvancedAnalyticsEvent('edit_alert_rule.incompatible_rule', {
```

Comment:
it seems like we could fire a lot of these

Reply:
Is there a way to sample? Or maybe should this only fire if there wasn't an incompatibility before?

Reply:
maybe send it once? you can use a property on the class